CorsixTH/Lua/world.lua

```diff
@@ -886,7 +760,7 @@ function World:setSpeed(speed)
   end
   if speed == "Pause" or self.system_pause then
     -- stop screen shaking if there was an earthquake in progress
-    if self.next_earthquake.active then
+    if self.earthquake:isActive() then
       self.ui:endShakeScreen()
```

Comment:
You moved the beginShakeScreen to PlayerHospital but not endShakeScreen. I don't have a strong preference for which they're in, but they should be in the same place.